react-native-scripts/template/README.md

```diff
@@ -120,3 +121,15 @@ If you have made use of Expo APIs while working on your project, then those API
 ## Tips and Tricks
 
 * If you have a local network the prevents your phone from accessing the address that is printed when running the packager, you can often work around this issue by "tethering" your computer to your phone's internet connection. Some phones call this feature "mobile hotspot." While it's very convenient, if you have a data plan where you pay for data transfer you should be careful when using this feature, especially when installing dependencies from NPM.
+
+## Troubleshooting
+
+* **QR code doesn't work when using Webstorm's built-in terminal**
```

Comment:
Hey, I should mention, WebStorm only has one light theme. And that's the default theme as well, and the only one which renders the QR code correctly. 

I'll work on your valuable suggestion, and try to carve the heading using the phrase you mentioned.